cmd/query/app/http_handler.go

```diff
@@ -481,14 +459,14 @@ func (aH *APIHandler) getTrace(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var uiErrors []structuredError
-	structuredRes := aH.tracesToResponse([]*model.Trace{trc}, shouldAdjust(r), uiErrors)
+	structuredRes := aH.tracesToResponse([]*model.Trace{trc}, uiErrors)
 	aH.writeJSON(w, r, structuredRes)
 }
 
-func shouldAdjust(r *http.Request) bool {
+func shouldUseRawTraces(r *http.Request) bool {
 	raw := r.FormValue("raw")
```

Comment:
Need to add tests with/without this arg. Also, since it's optional, should you be calling ParseBool unconditionally on it?

Reply:
@yurishkuro This was the existing behaviour. It looks like we ignored anything that wasn't true. Do we want to change that here?

Reply:
do we want the api to return an error here if the parameter isn't explicitly `true` or `false`?

Reply:
I think it's ok to be flexible similar to https://pkg.go.dev/strconv#ParseBool. I.e. `t` or `1` are fine, but `fooobar` should return an error.